apps/ledger-live-mobile/src/screens/Settings/General/WalletSyncRow.tsx

```diff
@@ -22,10 +23,12 @@ const WalletSyncRow = () => {
 
   const isDrawerVisible = useSelector(activateDrawerSelector);
   const dispatch = useDispatch();
+  const { setCurrentStep } = useCurrentStep();
 
   const closeDrawer = useCallback(() => {
     dispatch(setLedgerSyncActivateDrawer(false));
-  }, [dispatch]);
+    setCurrentStep(Steps.Activation);
```

Comment:
It's the only file that we need to do that?

Reply:
Yes, it's the only place where we set it to false